dashboard/test/ui/features/applab/scenarios.feature

```diff
@@ -16,7 +16,7 @@ Feature: App Lab Scenarios
   @no_mobile
   Scenario: App Lab Share
     # Create an app with a "hello world" button (omitting closing paren for auto-correct).
-    Then I press "show-code-header"
```

Comment:
is there any behavior difference for Then and When or is this just syntactic sugar to make it read better?